libraries/AP_Airspeed/Airspeed_Calibration.cpp

```diff
@@ -39,7 +39,7 @@ void Airspeed_Calibration::init(float initial_ratio)
   update the state of the airspeed calibration - needs to be called
   once a second
  */
-float Airspeed_Calibration::update(float airspeed, const Vector3f &vg)
+float Airspeed_Calibration::update(float aspeed, const Vector3f &vg)
```

Comment:
I tried using _airspeed_ here but it wouldn't compile - failed due to re-definition of _airspeed_ if I remember correctly.  I tried working out a way of fixing it so it would use _airspeed_ but couldn't and this ended up working.